internal/sign/job/signer.go

```diff
@@ -80,6 +89,14 @@ func (m *signer) MakeJobTemplate(
 		args = append(args, "-filestosign", strings.Join(signConfig.FilesToSign, ":"))
 	}
 
+	if signConfig.UnsignedImageRegistryTLS.Insecure {
```

Comment:
why is sign TLS logic different from build? they both can push images. Why we don't need to determine registryTLS just as we did in the build?

Reply:
They are not different. Same stuff are done in sign and build.